packages/reusable-blocks/src/store/test/actions.js

```diff
@@ -1,17 +1,271 @@
+	registry.register( reusableBlocksStore );
+	registry.register( blocksStore );
+
+	// Register entity here instead of mocking API handlers for loadPostTypeEntities()
+	registry.dispatch( coreStore ).addEntities( [
+		{
+			baseURL: '/wp/v2/reusable-blocks',
+			kind: 'postType',
+			name: 'wp_block',
+			label: 'Reusable blocks',
+		},
+	] );
+
+	return registry;
+}
 
 describe( 'Actions', () => {
+	beforeAll( () => {
+		registerBlockType( 'core/test-block', {
+			title: 'Test block',
+			category: 'text',
+			save: () => null,
+			attributes: {
+				name: { type: 'string' },
+			},
+		} );
+
+		registerBlockType( 'core/block', {
+			title: 'Reusable block',
+			category: 'text',
+			save: () => null,
+			attributes: {
+				ref: { type: 'string' },
+			},
+		} );
+	} );
+
+	afterAll( () => {
+		unregisterBlockType( 'core/test-block' );
+		unregisterBlockType( 'core/block' );
+	} );
+
 	describe( '__experimentalSetEditingReusableBlock', () => {
-		it( 'should return the SET_EDITING_REUSABLE_BLOCK action', () => {
-			const result = __experimentalSetEditingReusableBlock( 3, true );
-			expect( result ).toEqual( {
-				type: 'SET_EDITING_REUSABLE_BLOCK',
-				clientId: 3,
-				isEditing: true,
+		it( 'should flip the editing state', () => {
+			const registry = createRegistryWithStores();
+
+			registry
+				.dispatch( reusableBlocksStore )
+				.__experimentalSetEditingReusableBlock( 3, true );
+			expect(
+				registry
+					.select( reusableBlocksStore )
+					.__experimentalIsEditingReusableBlock( 3 )
+			).toBe( true );
+
+			registry
+				.dispatch( reusableBlocksStore )
+				.__experimentalSetEditingReusableBlock( 3, false );
+			expect(
+				registry
+					.select( reusableBlocksStore )
+					.__experimentalIsEditingReusableBlock( 3 )
+			).toBe( false );
+		} );
+	} );
+
+	describe( '__experimentalConvertBlocksToReusable', () => {
+		it( 'should convert a static block into a reusable block', async () => {
+			const staticBlock = createBlock( 'core/test-block', {
+				name: 'Big Bird',
 			} );
+
+			const registry = createRegistryWithStores();
+			// mock the api-fetch
+			apiFetch.mockImplementation( async ( args ) => {
+				const { path, data } = args;
+				switch ( path ) {
+					case '/wp/v2/reusable-blocks':
+						return {
+							id: 'new-id',
+							...data,
+						};
+					default:
+						throw new Error( `unexpected API endpoint: ${ path }` );
+				}
+			} );
+
+			registry.dispatch( blockEditorStore ).insertBlock( staticBlock );
+
+			await registry
+				.dispatch( reusableBlocksStore )
+				.__experimentalConvertBlocksToReusable( [
+					staticBlock.clientId,
+				] );
+
+			// check that blocks were converted to reusable
+			const updatedBlocks = registry
+				.select( blockEditorStore )
+				.getBlocks();
+
+			const newClientId = updatedBlocks[ 0 ].clientId;
+
+			expect( updatedBlocks ).toHaveLength( 1 );
+
+			// Delete random clientId before matching with snapshot
+			delete updatedBlocks[ 0 ].clientId;
```

Comment:
I had a similar issue in another test and decided to do this:
```js
updatedBlocks[ 0 ].clientId = `[${ typeof updatedBlocks[ 0 ].clientId }]`
```

At least the test can assert the type of the property is correct. Not something required, but just thought I'd share.